src/pages/Patient/components/PatientModal.tsx

```diff
@@ -0,0 +1,248 @@
+import { useCreatePatientMutation } from "@api";
+import { VisuallyHiddenInput } from "@elements";
+import {
+  Box,
+  Button,
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  InputLabel,
+  Modal,
+  OutlinedInput,
+  Typography,
+} from "@mui/material";
+import { matchIsValidTel, MuiTelInput } from "mui-tel-input";
+import { ChangeEvent, FC, FormEvent, Fragment, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import CloudUploadIcon from "@mui/icons-material/CloudUpload";
+
+type PatientModalProps = {
+  open: boolean;
+  onUpdate: () => void;
+  onClose: () => void;
+};
+
+export const PatientModal: FC<PatientModalProps> = ({
+  open,
+  onUpdate,
+  onClose,
+}) => {
+  const { t } = useTranslation("", { keyPrefix: "pages.patient" });
+  const [patientForm, setCabinetForm] = useState<{
+    name: string;
+    secondName: string;
```

Comment:
Move  this type  out  form  component  . Can be  described   under PatientModalProps